pkg/skaffold/debug/cnb.go

```diff
@@ -80,6 +80,10 @@ func updateForCNBImage(container *v1.Container, ic imageConfiguration, transform
 	}
 
 	c, img, err := transformer(container, ic)
+	// must explicitly modify the working dir as the imageConfig is lost after we return
+	if c.WorkingDir == "" {
```

Comment:
Seems a bit weird that you'd have to modify the object immediately after creating it. Is this only happening when `err != nil`? Should the default value be rather set in `transform.go` line 344:
```
return ContainerDebugConfiguration{ WorkingDir: config.workingDir }, "", fmt.Errorf("unable to determine runtime for %q", container.Name)
```

Reply:
nevermind, it got merged while I was looking at it 😄

Reply:
Also I totally skipped the description which explains it 😅